activate.sh

```diff
@@ -71,28 +71,46 @@ fi
 # Make sure there is no ROOTSYS (which would confuse ROOT)
 unset ROOTSYS
 
-# We need to add the path to the ST library dir to the
-# path that ROOT will search for libraries, because ROOT
-# does not honor RPATH
+# Check whether the .rootrc file exists in the user home,
+# if not create it
+if [ -f "${HOME}/.rootrc" ]; then
+        
+    # Make it read/write
+    chmod u+rw ${HOME}/.rootrc
 
-cat << EOF | root -b -l
+else
+        
+    # File does not exist. Copy the system.rootrc file
+    cp ${CONDA_PREFIX}/etc/root/system.rootrc ${HOME}/.rootrc
+    
+    # Make it read/write
+    chmod u+rw ${HOME}/.rootrc
+
+fi
+
+# We need to make sure that the path to the ST library dir is
+# contained in the paths that ROOT will search for libraries, 
+# because the dynamic loader of ROOT does not honor RPATH
+
+cat <<- EOF | root -b -l
 
 TString old_value=gEnv->GetValue("Unix.*.Root.DynamicPath", "hey");
```

Comment:
Why is the default value "hey"?
Should this be the empty string instead?

Reply:
I was having problem with the empty string. But we could change it to something less ludicrous, like "undefined" or "empty"